content/en/blog/2023/sig-cli-spotlight.md

```diff
@@ -0,0 +1,159 @@
+
+**Katrina**: For major features, you can check out [our open
+KEPs](https://www.kubernetes.dev/resources/keps/). For instance, in
+1.27 we introduced alphas for [a new pruning mode in kubectl
+apply](https://kubernetes.io/blog/2023/05/09/introducing-kubectl-applyset-pruning/),
+and for kubectl create plugins. Exciting ideas that are currently
+under discussion include an interactive mode for `kubectl` delete
+([KEP
+3895](https://kubernetes.io/blog/2023/05/09/introducing-kubectl-applyset-pruning))
+and the `kuberc` user preferences file ([KEP
+3104](https://kubernetes.io/blog/2023/05/09/introducing-kubectl-applyset-pruning)).
+
+**Arpit**: Could you discuss any challenges that SIG CLI faces in its
+efforts to improve CLIs for cloud-native technologies? What are the
+future efforts to solve them?
+
+**Katrina**: The biggest challenge we’re facing with every decision is
+backwards compatibility and ensuring we don’t break existing users. It
+frequently happens that fixing what's on the surface may seem
+straightforward, but even fixing a bug could constitute a breaking
+change for some users, which means we need to go through an extended
+deprecation process to change it, or in some cases we can’t change it
+at all. Another challenge is the need to balance customization with
+usability in the flag sets we expose on our tools. For example, we get
+many proposals for new flags that would certainly be useful to some
+users, but not a large enough subset to justify the increased
+complexity having them in the tool entails for everyone. The `kuberc`
+proposal may help with some of these problems by giving individual
+users the ability to set or override default values we can’t change,
+and even create custom subcommands via aliases
+
+**Arpit**: With every new version release of Kubernetes, maintaining
+consistency and integrity is surely challenging: how does the SIG CLI
+team tackle it?
+
+**Maciej**: This is mostly similar to the topic mentioned in the
+previous question: every new change, especially to existing commands
+goes through a lot of scrutiny to ensure we don’t break existing
+users. At any point in time we have to keep a reasonable balance
+between features and not breaking users.
+
+## Future plans and contribution
+
+**Arpit**: How do you see the role of CLI tools in the cloud-native
+ecosystem evolving in the future?
+
+**Maciej**: I think that CLI tools were and will always be an
+important piece of the ecosystem. Whether used by administrators on
+remote machines that don’t have GUI or in every CI/CD pipeline, they
+are irreplaceable.
+
+**Arpit**: Kubernetes is a community-driven project. Any
+recommendation for anyone looking into getting involved in SIG CLI
+work? Where should they start? Are there any prerequisites?
+
+**Maciej**: There are no prerequisites other than a little bit of free
+time on your hands and willingness to learn something new :-)
+
+**Katrina**: A working knowledge of Go often helps, but we also have
```

Comment:
(nit) we could link to https://go.dev/ here.

Reply:
Done!